tooling/src/hypothesistooling/__main__.py

```diff
@@ -306,6 +306,29 @@ def update_python_versions():
         build_sh.chmod(0o755)
 
 
+def update_pyodide_versions():
+    vers_re = r"(\d+\.\d+\.\d+)"
+    pyodide_version = re.findall(
+        f"pyodide_build-{vers_re}-py3-none-any.whl",
+        requests.get("https://pypi.org/simple/pyodide-build/").text,
+    )[-1]
+    makefile_url = f"https://raw.githubusercontent.com/pyodide/pyodide/{pyodide_version}/Makefile.envs"
+    python_version, emscripten_version = re.search(
+        rf"export PYVERSION \?= {vers_re}\nexport PYODIDE_EMSCRIPTEN_VERSION \?= {vers_re}\n",
+        requests.get(makefile_url).text,
+    ).groups()
```

Comment:
This looks brittle, but I suppose it's ok as it will be easy to figure out if it ever starts failing (and makefile syntax is notoriously struct)